src/pages/PageHome.vue

```diff
@@ -12,7 +12,7 @@ const links = [
     text: 'OSDay 2024',
   },
   {
-    href: 'https://opencollective.com/schrodinger-hat',
+    href: 'https://opencollective.com/schroedinger-hat',
```

Comment:
We can't update the opencollective url

```suggestion
    href: 'https://opencollective.com/schrodinger-hat',
```